sdk/spring/spring-reference.yml

```diff
@@ -4,1091 +4,1091 @@
 ---
 - name: Spring Cloud Azure Bom
   content:
-    - name: Spring Cloud Azure Bom
```

Comment:
Let's keep the indent same to the upstream:
https://github.com/saragluna/spring-reference-intellij/pull/11/files
![image](https://user-images.githubusercontent.com/13167207/230393571-971725f8-032f-4dee-8980-2f2df8d5d81f.png)